src/app/open-api-generator/api-generator.ts

```diff
@@ -244,13 +249,18 @@ export async function generateOpenApiTypescriptFile(
         }
       },
       onInit: (configuration) => {},
-      onPreParseSchema: (originalSchema, typeName, schemaType) => {},
+      onPreParseSchema: (originalSchema, typeName, schemaType) => {
+        // console.log('\n\n\n\nonPreParseSchema;; originalSchema ', CircularJSON.stringify(originalSchema));
+        originalSchema.description = OpenApiParser.fixDescription(
+          originalSchema.description,
+        );
+        // originalSchema.example = OpenApiParser.fixTypesDescriptions(originalSchema, schemaType, typeName);
+        return originalSchema;
+      },
```

Comment:
nit: Let's remove the stray comments?

Reply:
The only reason why I've left them in is because many a times while investigating an issues, these logs are what I need to look at. Given that this is still in alpha and debugging/refactoring is required frequently right now, I wanted to leave these in so that I can save time on having to write them again. Do you think it's okay to leave them in for now and remove them when we're about to go Beta at least (currently, we're in alpha)